src/hooks/useSearchField/index.ts

```diff
@@ -92,8 +92,6 @@ export default function useSeachField(flowID: string, runNumber: string): Search
         updateSearchResults({ result: event.matches || [], status: 'Ok' });
       } else if (event.type === 'error' && event.message) {
         updateSearchResults({ status: 'Error', errorMsg: event.message, result: [] });
-      } else {
-        updateSearchResults({ result: [], status: 'Ok' });
```

Comment:
so was the issue that for `event.type === 'progress'` the loading would stop? Looks like a good change.

Reply:
Yep. We could try to figure out how to utilise that progress message